src/programmatic-dmpu/web/index.scss

```
@@ -0,0 +1,28 @@
+@import '_core';
+
+.creative--programmatic-dmpu {
+    margin: 0 auto;
+    width: 300px;
+    height: 600px;
+    top: 0;
+    left: 0;
+    position: absolute;
+}
+
+.creative__image {
+  margin: 0 auto;
+  width: 300px;
+  height: 600px;
+  top: 0;
+  left: 0;
+  position: absolute;
+}
+
+.creative__link {
+  margin: 0 auto;
+  width: 300px;
+  height: 600px;
+  top: 0;
+  left: 0;
+  position: absolute;
+}
```

If you’re applying the same styles, here’s a shorter declaration

```suggestion
.creative--programmatic-dmpu,
.creative__image,
.creative__link
 {
    margin: 0 auto;
    width: 300px;
    height: 600px;
    top: 0;
    left: 0;
    position: absolute;
}
```